File: 4-median-of-two-sorted-arrays/median-of-two-sorted-arrays.py
class Solution(object):
    def findMedianSortedArrays(self, nums1, nums2):
        """
        :type nums1: List[int]
        :type nums2: List[int]
        :rtype: float
        """

        nums1.extend(nums2)
        b=len(nums1)
        nums1.sort()
        if b%2==0:
            return (nums1[b//2]+nums1[b//2-1])/2.0
        else:
            return nums1[b//2]

        

File: 4-median-of-two-sorted-arrays/median-of-two-sorted-arrays.java
class Solution {
    public double findMedianSortedArrays(int[] nums1, int[] nums2) {
        int[] arr = new int[nums1.length+nums2.length];
        int b = arr.length;
        for(int i=0;i<nums1.length;i++){
            arr[i]=nums1[i];
        }
        int j=0;
        for(int i=nums1.length;i<nums1.length+nums2.length;i++){
            arr[i]=nums2[j];
            j++;
        }
        Arrays.sort(arr);

        if(b%2==0){
            return (arr[b/2]+arr[b/2-1])/2.0;
        }
        else{
            return arr[b/2];
        }
    }
}